.dockerignore

```diff
@@ -2,9 +2,10 @@
 .*
 
 # Top-level directories unrelated to the build.
```

Comment:
We could also add `.github/` here, right? Also, what about `infrastructure/`, `scripts/` - could we add them here as well?

Reply:
We already have them.

`.github` should be covered by 

https://github.com/keep-network/keep-core/blob/a2ae4270fe2446fa1fabf44398326fbbc88dc4d6/.dockerignore#L1-L2

`infrastructure/` and `scripts/` are here:
https://github.com/keep-network/keep-core/blob/a2ae4270fe2446fa1fabf44398326fbbc88dc4d6/.dockerignore#L6-L7

Reply:
Gosh, I'm blind 🙈